src/components/AutocompleteMnxReaction.vue

```diff
@@ -388,48 +315,10 @@ export default Vue.extend({
           this.selectedItem = null;
         });
       }
-      const reaction: Reaction = {
-        id: selectedReaction.foundId || selectedReaction.reaction.mnx_id,
-        name: selectedReaction.reaction.name || "",
-        // TODO: rebuild reaction string more consistently, from equation_parsed
-        reactionString: this.equationDisplay(selectedReaction),
-        // Note: Assuming all reactions in the universal model are
-        // reversible, but this might not be the case. Could potentially use
-        // the reaction string to check reversibility.
-        lowerBound: -1000,
-        upperBound: 1000,
-        metabolites: selectedReaction.reaction.equation_parsed.map(m => {
-          const fullMetabolite = selectedReaction.metabolites.find(
-            ({ mnx_id }) => mnx_id === m.metabolite_id
-          );
-          return {
-            id: m.metabolite_id,
-            name: fullMetabolite ? fullMetabolite.name : "",
-            formula: fullMetabolite ? fullMetabolite.formula : "",
-            // TODO: use m.compartment_id, mapped through selectedReaction.annotations
-            compartment: "",
-            stoichiometry: m.coefficient
-          };
-        }),
-        namespace: selectedReaction.namespace
-      };
-
-      this.$emit("change", {
-        reaction: reaction,
-        mnxReaction: selectedReaction
```

Comment:
Issue: we are using it in `CardDialogDesign/addReaction` method

Reply:
Thanks for the catch!